source/common/network/utility.cc

```diff
@@ -132,14 +132,14 @@ Address::InstanceConstSharedPtr Utility::parseInternetAddressNoThrow(const std::
   if (inet_pton(AF_INET, ip_address.c_str(), &sa4.sin_addr) == 1) {
     sa4.sin_family = AF_INET;
     sa4.sin_port = htons(port);
-    return std::make_shared<Address::Ipv4Instance>(&sa4);
+    return *Address::createInstance<Address::Ipv4Instance>(&sa4);
```

Comment:
propagate errors up.

Reply:
There is no potential error here, this Instance constructor always succeed.

Reply:
Can you use RELEASE_ASSERT in such cases?

Reply:
Thanks, will fix.